### PR TITLE
Section deploying R to production

### PR DESCRIPTION
**Pull request by nuest**

### CODE REVIEW

File: manuscript.Rmd [hunk 1]
```diff
@@ -302,8 +308,95 @@ The cloud is the natural environment of containers, and becomes the go-to mechan
...
+
+R has been historically viewed as a tool for analysis and scientific research--not for creating software that corporations can rely on to continously run in real time. However, thanks to advancements in R running as a web service, along with with the ability to deploy R in docker containers, modern enterprises are now capable of having real-time machine learning powered by R.
+
+The backbone of this work is the _[plumber](https://www.rplumber.io/docs/hosting.html#docker)_ package, created by Jeff Allen. The plumber package allows R to be used to host web services through RESTful APIs. When R is running on a server, a different server can ask R to make a computation through an HTTP request and get an immediate response. For instance, if a data scientist at a company creates a machine learning model to tell which customers are likely to make a repeat purchase, if they serve the model through a plumber API then someone else on the marketing team could write software that sends different emails depending on that real-time prediction. Plumber syntax is simple to read and understand: the API is created by commenting above function names in an R script.
```
Comment (nuest) [inline manuscript.Rmd]: We have to keep an eye on the level of detail, and to me this goes a bit beyond the information about plumber that we can squeeze in. I'll try to shorten it.

File: manuscript.Rmd [hunk 2]
```diff
@@ -302,8 +308,95 @@ The cloud is the natural environment of containers, and becomes the go-to mechan
...
+
+At T-Mobile, R is used to power real-time services that are called over a million times a day. T-Mobile created a set of neural network machine learning natural language processing models to help customer care agents manage text-based messages for customers. The models are convolutional neural networks that use the RStudio Keras package and are built on top of a Rocker Docker image. Since the models power tools for agents and customers, they need to have extremely high uptime and reliability--and they found that R is able to perform. The AI @ T-Mobile team open sourced [their docker images](https://github.com/tmobile/r-tensorflow-api  which are available on GitHub.
```
Comment (nuest) [inline manuscript.Rmd]: This is a great story!

I'll add a reference to https://opensource.t-mobile.com/blog/posts/r-tensorflow-api/ after the merge.

File: manuscript.Rmd [hunk 3]
```diff
@@ -302,8 +308,95 @@ The cloud is the natural environment of containers, and becomes the go-to mechan
...
+
+While these load-balancing and autoscaling properties of a Kubernetes cluster are invaluable, they are still limited by the hardware resources that it has access to. A cluster cannot scale out if it has run out of servers. This issue in turn can be addressed by deploying not to on-premise servers but to the cloud, where effectively unlimited computing resources are available (as long as one can pay for them). From a management perspective, cloud deployments have the additional advantage that hardware maintenance becomes the responsibility of the cloud provider, eliminating another pain point. Orchestrator services are available from all the major cloud service providers, including Amazon Web Services (AWS), Google Cloud Platform, and Microsoft Azure.
```
Comment (nuest) [inline manuscript.Rmd]: Same here: This is all helpful, but potentially a bit too extensive for the scope of the article. I'll work on a revision and let you know.

**Comment by nuest** [comment]
@Hong-Revo @jnolis Thanks for your contributions!

Keeping in mind the goal of the article, these are very interesting sections for the reader. I'll make another iteration and see if I can shorten some sections providing background, potentially find good references for it.

@jnolis could you please review you affiliation details:  Thanks!

**Comment by jnolis** [comment]
Just fixed my affiliations details and added @nolistic too (who helped me with the T-Mobile part).

**Pull request merged**